app/controllers/user/bookings_controller.rb

```diff
@@ -49,4 +55,21 @@ def load_football_pitch
     flash[:error] = t("football_pitch.not_found")
     redirect_to root_path
   end
+
+  def current_user_booking_pitches
+    @booking_football_pitches = Booking.where(id: current_user.id)
+    return if @booking_football_pitches
+
+    flash[:error] = t "football_pitches.find_pitch_failed"
+    redirect_to root_path
+  end
+
+  def search_booked_pitch
+    booked_pitch = params[:football_pitch_name]
+    return if booked_pitch.blank?
+
+    @booking_football_pitches = Booking
```

Comment:
```suggestion
    @booking_football_pitches = current_user.bookings.search_by_booked_pitch_name(booked_pitch)
```

ưu tiên gọi qua association nhé, e đặt quan hệ trong các model là gọi dc